evennia/commands/default/help.py

```diff
@@ -228,14 +229,35 @@ def func(self):
         self.msg(self.format_help_entry("", "No help entry found for '%s'" % query, None, suggested=suggestions))
 
 
+def loadhelp(caller):
```

Comment:
These should probably be made into private functions (starting with `_`); then they also don't need the full docstrings (which would not help readability in this case).